x-pack/plugins/data_enhanced/server/search/session/session_service.test.ts

```diff
@@ -176,92 +175,92 @@ describe('SearchSessionService', () => {
-    const mockDeps = {} as SearchSessionDependencies;
-
-    beforeEach(() => {
-      mockSearch.mockClear();
-    });
-
-    it('searches using the original request if not restoring', async () => {
-      const searchRequest = { params: {} };
-      const options = { sessionId, isStored: false, isRestore: false };
-
-      await service
-        .search(mockStrategy, searchRequest, options, mockSearchDeps, mockDeps)
-        .toPromise();
-
-      expect(mockSearch).toBeCalledWith(searchRequest, options, mockSearchDeps);
-    });
-
-    it('searches using the original request if `id` is provided', async () => {
-      const searchId = 'FnpFYlBpeXdCUTMyZXhCLTc1TWFKX0EbdDFDTzJzTE1Sck9PVTBIcW1iU05CZzo4MDA0';
-      const searchRequest = { id: searchId, params: {} };
-      const options = { sessionId, isStored: true, isRestore: true };
-
-      await service
-        .search(mockStrategy, searchRequest, options, mockSearchDeps, mockDeps)
-        .toPromise();
-
-      expect(mockSearch).toBeCalledWith(searchRequest, options, mockSearchDeps);
-    });
-
-    it('searches by looking up an `id` if restoring and `id` is not provided', async () => {
-      const searchRequest = { params: {} };
-      const options = { sessionId, isStored: true, isRestore: true };
-      const spyGetId = jest.spyOn(service, 'getId').mockResolvedValueOnce('my_id');
-
-      await service
-        .search(mockStrategy, searchRequest, options, mockSearchDeps, mockDeps)
-        .toPromise();
-
-      expect(mockSearch).toBeCalledWith({ ...searchRequest, id: 'my_id' }, options, mockSearchDeps);
-
-      spyGetId.mockRestore();
-    });
-
-    it('calls `trackId` once if the response contains an `id` and not restoring', async () => {
-      const searchRequest = { params: {} };
-      const options = { sessionId, isStored: false, isRestore: false };
-      const spyTrackId = jest.spyOn(service, 'trackId').mockResolvedValue();
-      mockSearch.mockReturnValueOnce(of({ id: 'my_id' }, { id: 'my_id' }));
-
-      await service
-        .search(mockStrategy, searchRequest, options, mockSearchDeps, mockDeps)
-        .toPromise();
-
-      expect(spyTrackId).toBeCalledTimes(1);
-      expect(spyTrackId).toBeCalledWith(searchRequest, 'my_id', options, {});
-
-      spyTrackId.mockRestore();
-    });
-
-    it('does not call `trackId` if restoring', async () => {
-      const searchRequest = { params: {} };
-      const options = { sessionId, isStored: true, isRestore: true };
-      const spyGetId = jest.spyOn(service, 'getId').mockResolvedValueOnce('my_id');
-      const spyTrackId = jest.spyOn(service, 'trackId').mockResolvedValue();
-      mockSearch.mockReturnValueOnce(of({ id: 'my_id' }));
-
-      await service
-        .search(mockStrategy, searchRequest, options, mockSearchDeps, mockDeps)
-        .toPromise();
-
-      expect(spyTrackId).not.toBeCalled();
-
-      spyGetId.mockRestore();
-      spyTrackId.mockRestore();
-    });
-  });
+  // describe('search', () => {
```

Comment:
Why are these commented out?

Reply:
I need to update these since we're no longer exposing a `search` method on the search service to have coverage over `getSearchIdMapping` and `getId`.